src/store/selectors/index.js

```diff
@@ -1,36 +1,42 @@
 import { createSelector } from 'reselect'
-import { mapValues, every } from '../../utils/func.js'
-import { createSelectorWhichMapsToHash } from './utils.js'
-
-const captures = state => state.captures
-const createSelectorWhichMapsToCaptures = (mapFunc) =>
-        createSelectorWhichMapsToHash(captures, mapFunc)
-
-export const isThereAValidCapture = createSelectorWhichMapsToCaptures(capturesOfAType =>
-  capturesOfAType.some(i => i.valid))
-
-export const isThereAValidAndConfirmedCapture = createSelectorWhichMapsToCaptures(capturesOfAType =>
-  capturesOfAType.some(i => i.valid && i.confirmed))
-
-export const validCaptures = createSelectorWhichMapsToCaptures(capturesOfAType =>
-  capturesOfAType.filter(i => i.valid))
-
-export const unprocessedCaptures = createSelectorWhichMapsToCaptures(capturesOfAType =>
-    capturesOfAType.filter(i => !i.processed))
+import mapValues from 'object-loops/map'
+
+const currentCaptures = (state, props) => {
+  const method = props.method
+  const side = props.side ? props.side : null
+  return state.captures[method].filter(c => c.side === side)
+}
+
+const outputCaptures = (state) => {
+  const captures = state.captures
+  const outputCaptures = {}
+  outputCaptures.face = captures.face
+  outputCaptures.document = captures.document.filter(c => c.side === 'front')
+  outputCaptures.documentBack = captures.document.filter(c => c.side === 'back')
+  return outputCaptures
+}
+
+export const currentValidCaptures = createSelector(
+  currentCaptures,
```

Comment:
reselect passes the arguments to all chained selectors so calling `currentValidCaptures(state, props)` will pass props to currentCaptures